pkg/apis/wsecurity/v1/README.md

```diff
@@ -0,0 +1,47 @@
+# Security Data Package
+
+This package serves as the beating heart of Guard.
```

Comment:
Do you want to link to a design or architecture document?

Reply:
going forward, yes.